travis.sh

```diff
@@ -22,10 +22,10 @@ cmp /tmp/initramfs.linux_amd64.cpio /tmp/i2
  which go
 
 echo "-----------------------> First ramfs test"
- (cd scripts && go run ramfs.go -tmpdir=/tmp/u-root -removedir=false)
+ (cd scripts && go run ramfs.go -tmpdir=/tmp/u-root)
 
 echo "-----------------------> Build ip command"
- GOBIN=/tmp/u-root/ubin GOROOT=/tmp/u-root/go GOPATH=/tmp/u-root CGO_ENABLED=0 /tmp/u-root/go/bin/go build  github.com/u-root/u-root/cmds/ip
+# GOBIN=/tmp/u-root/ubin GOROOT=/tmp/u-root/go GOPATH=/tmp/u-root CGO_ENABLED=0 go build  github.com/u-root/u-root/cmds/ip
```

Comment:
why is this commented out?

Reply:
/tmp/u-root doesn't actually contain any of those files anymore (go compiler, or the ip command), since the script now doesn't copy all files into a temporary directory anymore.

Reply:
oh whoops just remove it then? It's a bit confusing.